File: Data Structures & Algorithms/kth-largest-element-in-an-array/submission-0.py
class Solution:
    def findKthLargest(self, nums: List[int], k: int) -> int:
        
        heapq.heapify_max(nums)
        print(nums)
        x = k

        while x > 1:
            heapq.heappop_max(nums)
            x -= 1
        
        return heapq.heappop_max(nums)

File: Data Structures & Algorithms/move-zeroes/submission-0.java
class Solution {
    public void moveZeroes(int[] nums) {
        
        for(int i=0; i <nums.length - 1; i++){
            int right = i+1;
            if (nums[i] == 0){
                while(right < nums.length - 1 && nums[right] == 0){
                    right++;
                }
                int tmp = nums[right];
                nums[right] = nums[i];
                nums[i] = tmp;
            } 
        }
    }
}